modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/searcher/SearchRequestContributorsRegistryImpl.java

```diff
@@ -99,7 +82,7 @@ private Collection<SearchRequestContributor> _include(
 		Collection<String> ids) {
 
 		if ((ids == null) || ids.isEmpty()) {
-			return new ArrayList<>(_searchRequestContributors);
+			return _serviceTrackerList.toList();
```

Comment:
@W2jx Please remove service tracker list, collect the results by iterate the map's values.